srcpkgs/python3-paramiko/template

```diff
@@ -4,7 +4,8 @@ version=2.11.0
 revision=2
```

Comment:
Please increase the revision number.

The package must be rebuilt to let the missing dependency appear in the metadata.

Reply:
Fixed.